### PR TITLE
[WebProfilerBundle] Show profiler in full screen

### PR DESCRIPTION
**Pull request by peter-gribanov**
| Q | A |
| --- | --- |
| Branch? | master |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| License | MIT |

### CODE REVIEW

File: src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig [hunk 1]
```diff
@@ -402,7 +402,6 @@ tr.status-warning td {
 {# Layout
    ========================================================================= #}
 .container {
-    max-width: 1300px;
```
Comment (xabbuh) [inline src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/profiler.css.twig]: IIRC this was done on purpose to ensure readability.

**Comment by javiereguiluz** [comment]
👎 

This has been asked once in the past. We don't want to make this change because, even if it could be useful in some edge cases, most of the times this makes the profiler information harder to understand. It's a fact that very long lines destroy their readability (personally I even think that the current value is too high).

That's why I'm closing this proposal as "won't fix". I hope you understand it. Thanks!

**Comment by peter-gribanov** [comment]
For really large screens it is inconvenient
progiler

**Comment by javiereguiluz** [comment]
@peter-gribanov yes, that's the only case where it's useful: the timeline of complex requests. I had an idea to improve this but I don't have the time to implement it: show an icon called `expand ->` on each timeline chart. If you click on it, the chart expands to the full window width, whatever it is.

**Comment by peter-gribanov** [comment]
@javiereguiluz the main thing that you will agree that in some situations it is necessary :blush:

**Comment by javiereguiluz** [comment]
@peter-gribanov yes, in some edge cases this is useful ... but in 95% of the cases is harmful.

**Pull request closed without merging**